core/js/oc-backbone.js

```
@@ -11,4 +11,7 @@
 /* global Backbone */
 if(!_.isUndefined(Backbone)) {
 	OC.Backbone = Backbone.noConflict();
+	if(!_.isUndefined(Backbone)) {
```

if it's undefined the line above will fail hard 🙈